infra/compare.py

```diff
@@ -131,6 +131,7 @@ def compare_files(book, code, language, file):
                 print(f"{block.loc}:", error)
                 failure += 1
             continue
+        if "output" in block.classes: continue
```

Comment:
don't we need a change to book.css?

Reply:
Yes—I was planning to do all the style updates in a separate PR but I can do it all in this one if you'd like. Or just make the minimal change in this PR and restyle in another.

Reply:
style in another PR is fine